policy.go

```diff
@@ -529,28 +685,60 @@ func ensureSufficientORDigests(digests tpm2.DigestList) tpm2.DigestList {
 	return digests
 }
 
+// computePcrPolicyRefFromCounterName computes the reference used for authorization of signed PCR policies from the supplied
+// PCR policy counter name. If name is empty, then the name of the null handle is assumed. The policy ref serves 2 purposes:
+// 1) It limits the scope of the signed policy to just PCR policies (the dynamic authorization policy key may be able to sign
+//    different types of policy in the future, for example, to permit recovery with a signed assertion.
+// 2) It binds the name of the PCR policy counter to the static authorization policy.
+func computePcrPolicyRefFromCounterName(name tpm2.Name) tpm2.Nonce {
+	if len(name) == 0 {
+		name = make(tpm2.Name, binary.Size(tpm2.Handle(0)))
+		binary.BigEndian.PutUint32(name, uint32(tpm2.HandleNull))
+	}
+
+	h := tpm2.HashAlgorithmSHA256.NewHash()
+	h.Write([]byte("AUTH-PCR-POLICY"))
+	h.Write(name)
+
+	return h.Sum(nil)
+}
+
+// computePcrPolicyRefFromCounterContext computes the reference used for authorization of signed PCR policies from the supplied
+// ResourceContext.
+func computePcrPolicyRefFromCounterContext(context tpm2.ResourceContext) tpm2.Nonce {
+	var name tpm2.Name
+	if context != nil {
+		name = context.Name()
+	}
+
+	return computePcrPolicyRefFromCounterName(name)
+}
+
 // computeStaticPolicy computes the part of an authorization policy that is bound to a sealed key object and never changes.
```

Comment:
I didn't remark this before, but I think it would make sense to describe what these static policies are concretely